components/list/test/list-item-drag-handle.visual-diff.js

```diff
@@ -0,0 +1,45 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-list-item-drag-handle', () => {
+
+	const visualDiff = new VisualDiff('list-item-drag-handle', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await visualDiff.createPage(browser);
+		await page.goto(`${visualDiff.getBaseUrl()}/components/list/test/list-item-drag-handle.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+		await visualDiff.resetFocus(page);
```

Comment:
Normally the focus reset is done between tests if focus is applied during tests:
```
beforeEach(async() => {
	await visualDiff.resetFocus(page);
});
```
Currently, the `resetFocus` would be called before all these tests run, and only once.  It's not really doing anything for the tests.  For the current tests, focus is applied in the 2nd test, and would remain focused for the 3rd test, which makes its own focus call moot.